src/ore/src/retry.rs

```diff
@@ -136,6 +137,16 @@ impl Retry {
         self
     }
 
+    /// Sets the jitter backoff factor for the retry operation.
+    ///
+    /// The time to wait is varied by `+-jitter` after each failed try. The default is no jitter.
```

Comment:
note: the default is no jitter because otherwise we'd have to include `rand` as a dependency if we wanted a default, which I'm more than happy to do, but I'm not sure if we want to add another dependency to the `async` feature